tests/console/yast2_clone_system.pm

```diff
@@ -38,7 +38,27 @@ sub run {
 
     # Check and upload profile for chained tests
     upload_asset "/root/autoinst.xml";
-    if (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml') {
+
+    unless (is_opensuse) {
+        my $sle15_repo;
+        if (is_sle('>15')) {
+            my $sp = (split(/-/, get_var('VERSION')))[-1];
+            $sle15_repo = "REPO_SLE15_" . "$sp" . "_MODULE_DEVELOPMENT_TOOLS";
+        }
+        else {
+            $sle15_repo = 'REPO_SLE15_MODULE_DEVELOPMENT_TOOLS';
+        }
+
+        my $devel_repo = get_required_var(is_sle('>=15') ? $sle15_repo : 'REPO_SLE_SDK');
+        assert_script_run "zypper ar -c $utils::OPENQA_FTP_URL/" . $devel_repo . " devel-repo";
+    }
+
+    zypper_call '--gpg-auto-import-keys ref';
+    zypper_call 'install jing';
+
+    if ((script_run 'jing -c /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml')
+        || (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml')) {
+        record_soft_failure 'bsc#1104632';
```

Comment:
We need to put it in some condition, as for SLE15 test will die and hence fail instead of soft-failure.

Reply:
right, ty